app/mesh/changelog.md

```diff
@@ -4,13 +4,23 @@ no_version: true
 ---
 
 ## 1.9.0
-> Release on **TBD**
+> Release on **2022/08/22**
 
 Built on top of [Kuma 1.8.0](https://github.com/kumahq/kuma/releases/tag/1.8.0)
 
 ### Changes
 
-- TO DO
+- Add "replace" function to CommonName template in CAs which support it (ACMPCA, cert-manager, Vault).
+- Improve error message when non-https URL is used with OPA.
```

Comment:
I don't think it's important enough to have it in this changelog to be honest

Reply:
removed